editoast/src/tests/small_infra/small_infra.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "3.3.1",
+    "version": "3.6.0",
```

Comment:
This will need a rebase once https://github.com/osrd-project/osrd/pull/4728 is merged